doc/guides/device_mgmt/smp_groups/smp_group_2.rst

```diff
@@ -0,0 +1,170 @@
+.. _mcumgr_smp_group_2:
+
+Statistics management
+#####################
+
+Statistics management allows to obtain data gathered by Statistics subsystem
+of Zephyr, enabled by ``CONFIG_STATS``.
+
+Statistics management group defines commands:
+
+.. table::
+    :align: center
+
+    +-------------------+-----------------------------------------------+
+    | ``Command ID``    | Command description                           |
+    +===================+===============================================+
+    | ``0``             | Group data                                    |
+    +-------------------+-----------------------------------------------+
+    | ``1``             | List groups                                   |
+    +-------------------+-----------------------------------------------+
+
+Statistics: group data
+**********************
+
+The command is used to obtain data for group specified by a name.
+The name is one of group names as registered, with ``STATS_INIT_AND_REG`` macro
+or ``stats_init_and_reg(...)`` function call, within module that gathers
+the statistics.
+
+Statistics: group data request
+==============================
+
+Statistics group data request header:
+
+.. table::
+    :align: center
+
+    +--------+--------------+----------------+
+    | ``OP`` | ``Group ID`` | ``Command ID`` |
+    +========+==============+================+
+    | ``0``  | ``2``        |  ``0``         |
+    +--------+--------------+----------------+
+
+CBOR Payload of request:
+
+.. code-block:: none
+
+    {
+        (str)"name" :  (str)
+    }
+
+where:
+
+.. table::
+    :align: center
+
+    +-----------------------+---------------------------------------------------+
+    | "name"                | is group name                                     |
+    +-----------------------+---------------------------------------------------+
+
+Statistics: group data response
+===============================
+
+Statistics group data response header:
+
+.. table::
+    :align: center
+
+    +--------+--------------+----------------+
+    | ``OP`` | ``Group ID`` | ``Command ID`` |
+    +========+==============+================+
+    | ``1``  | ``2``        |  ``0``         |
+    +--------+--------------+----------------+
+
+CBOR Payload of response:
+
+.. code-block:: none
+
+    {
+        (str)"name"     : (str)
+        (str)"fields"   : {
+            (str)<entry_name> : (uint)
+            ...
+        }
+        (str)"rc"       : (int)
```

Comment:
I was unaware of this restriction with zcbor/cddl-gen: https://github.com/NordicSemiconductor/zcbor/issues/176
If the plan is to move to zcbor, should these entries show the current layout or the layout that things would go with? At present, the layout is rc before name